baus/datasources.py

```diff
@@ -23,7 +23,7 @@
 # this is similar to the code for settings in urbansim_defaults
 @orca.injectable('run_setup', cache=True)
 def run_setup():
-    with open("../run_setup.yaml") as f:
```

Comment:
Yeah good idea, since I think that's right-- with this setup we ignore any `run_setup.yaml` in the code / model inputs, and the one that gets written out is essential.

Reply:
Just added that change.